packages/tsconfig-reference/copy/id/options/disableSourceOfProjectReferenceRedirect.md

```diff
@@ -0,0 +1,7 @@
+---
+display: "Nonaktifkan Pengalihan Referensi Proyek Sumber"
+oneline: "Gunakan berkas d.ts sebagai sumber kebenaran untuk perkakas antara batasan proyek gabungan"
+---
+
+Saat bekerja dengan [proyek TypeScript komposit](/docs/handbook/project-references.html), opsi ini menyediakan cara untuk kembali [kembali ke pra-3.7](/docs/handbook/release-notes/typescript-3-7.html#build-free-editing-with-project-references) perilaku di mana berkas d.ts digunakan sebagai batas antar modul.
```

Comment:
`cara untuk kembali` konteks versi bahasa Inggrisnya ini apakah Mas?

Reply:
versi bhs inggirsnya `provides a way to go`

Reply:
klo diterjemahin `cara untuk pergi`

Reply:
salah nerjemahin tadi mas, ternyata di dalam linknya udah ada kata `kembali`

Reply:
bagaimana menurut temen" udah sesuai?

Reply:
kembali kembali?

Reply:
jadi setelh di teliti lagi mas, itu artinya `cara untuk menuju`